index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en"></html>
+<html>
+    <head>
+        <html lang="en">
+            <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+  
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+         <meta charset="UTF-8">
+         <title>Number Facts</title>
+         <style>
+            #facts{
+                display:none;
+            }
+         </style>
+    </head>
+    <body class="bg-dark">
+<div class="container">
+    <div class="row">
+        <div class="col-md-6 mx-auto">
+            <div class="card bg-primary text-white mt-5 p-4">
+                <h1>Number Facts</h1 >
+                    <p>Enter a Number and get a random fact</p>
+                    <input type="number" class="form-control form-control-lg" id="numberInput" placeholder="Enter any number">
+            <div class="card-body" id="facts">
+                <h4 class="card-title">Number Facts</h4>
+                <p id="factText" class="card-text"></p>
+            </div>
+                </div>
+        </div>
+    </div>
+</div>
+
+ <script>
+
+let fact=document.querySelector('#facts') ;
+let factText=document.querySelector('#factText') ;
+let numberInput= document.querySelector('#numberInput');
+// using Ajax
+numberInput.addEventListener('input',getFactFetch);
+// function getFactAjax(){
+//    let number = numberInput.value;
+
+//    let xhr = new XMLHttpRequest();
+//    xhr.open('GET','http://numbersapi.com/'+ number);
+
+
+//    xhr.onload = function(){
+// if(this.status ==200 && number !=''){
+//     facts.style.display = 'block';
+//     factText.innerHTML=this.responseText;
+// }
+//    }
+//    xhr.send();
+// }
+ function getFactFetch(){
+     let number = numberInput.value;
+
+     fetch('http://numbersapi.com/'+ number)
+     .then(Response => Response.text())
```

Comment:
Generally variables in JavaScript don't start with an uppercase letter. You can see this has confused the GitHub syntax highlighter as well, as it is a different colour to normal variables